src/visGeometry/index.ts

```diff
@@ -718,8 +717,9 @@ class VisGeometry {
             const visAgent = this.visAgents[i];
             if (typeIds.includes(visAgent.agentData.type)) {
                 visAgent.setColor(
-                    this.getColorForTypeId(visAgent.agentData.type),
-                    this.getColorIdForTypeId(visAgent.agentData.type)
+                    this.colorHandler.getColorInfoForAgentType(
+                        visAgent.agentData.type
+                    )
```

Comment:
I see what you did here. Maybe this deserves a named type like AgentColorInfo { color: Color, id: number}